docs/lang/articles/basic/field.md

```diff
@@ -3,100 +3,157 @@ sidebar_position: 3
 ---
 
 # Fields
+Taichi fields are used to store data.
+In general, fields are global data containers that can be read and written from both the Python scope and the Taichi scope.
 
-Fields are **global** variables provided by Taichi. **Global** indicates that fields can be read/written from both the Python scope and the Taichi scope. A field can be considered as a multi-dimensional array of elements, and it can be either **dense** or **sparse**. Similar to a NumPy `ndarray` object, a field has a data type and a shape. Moreover, an element of a field can be a scalar, a **vector**, a **matrix**, or a **struct**.
+A field has its own data type and shape and can be considered as a multi-dimensional array of elements.
+An element of a field can be a **scalar**, a **vector**, a **matrix**, or a **struct**.
+The sparsity of a field element is **dense** by default, but it can also be **sparse**, as detailed described in [Sparse computation](/lang/articles/advanced/sparse).
```

Comment:
```suggestion
The sparsity of a field element is **dense** by default, but it can also be **sparse**, as detailed described in [Sparse spatial data structures](/lang/articles/advanced/sparse).
```

Reply:
`Sparse computation` would be changed afterward.